crates/net/network/src/network.rs

```diff
@@ -344,6 +328,32 @@ impl NetworkSyncUpdater for NetworkHandle {
     }
 }
 
+impl EthNetwork for NetworkHandle {
```

Comment:
I'd like to remove this trait.

I know the issue mentioned trait but I changed my mind, and would rather have the `fn transactions_handle` as a regular function on NetworkHandle first, before we think about moving all of these to a trait